aggregation_mode/src/backend/s3.rs

```diff
@@ -13,38 +16,92 @@ pub enum GetBatchProofsError {
 // needed to make S3 bucket work
 const DEFAULT_USER_AGENT: &str = "proof-aggregator/aligned-layer";
 
-pub async fn get_aligned_batch_from_s3(
+// Retry parameters for S3 requests
+/// Initial delay before first retry attempt (in milliseconds)
+const RETRY_MIN_DELAY_MILLIS: u64 = 500;
+/// Exponential backoff multiplier for retry delays
+const RETRY_FACTOR: f32 = 2.0;
+/// Maximum number of retry attempts
+const RETRY_MAX_TIMES: usize = 5;
+/// Maximum delay between retry attempts (in seconds)
+const RETRY_MAX_DELAY_SECONDS: u64 = 10;
+
+/// Timeout for Reqwest Client
+const REQWEST_TIMEOUT_SECONDS: Duration = Duration::from_secs(60);
```

Comment:
This is the timeout for downloading the batch right ? It may be a bit low, and it may be better to call it download batch timeout or something like that

```suggestion
const REQWEST_TIMEOUT_SECONDS: Duration = Duration::from_secs(60);
```

Reply:
Solved on #1149cae71eeb0aeb53cf51e5521e95aaabb36580